debian/control

```diff
@@ -0,0 +1,15 @@
+Source: imx-firmware
+Section: contrib/electronics
+Priority: optional
+Build-Depends: debhelper-compat (= 12)
+Standards-Version: 3.9.8
```

Comment:
Use a recent Standards-Version.
https://www.debian.org/doc//debian-policy/ch-controlfields.html#standards-version
https://www.debian.org/doc//debian-policy/index.html

Reply:
With version 4.6.1 there is a warning ... https://www.debian.org/doc/debian-policy/upgrading-checklist.html
That is the reason because I Updated it to 4.3.0